### PR TITLE
New version: GitHub.GitHubDesktop.Beta version 3.4.1-beta1

### PR DESCRIPTION
**Pull request by damn-good-b0t**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156498)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 883.)

File: manifests/g/GitHub/GitHubDesktop/Beta/3.4.1-beta1/GitHub.GitHubDesktop.Beta.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: GitHub.GitHubDesktop.Beta
+PackageVersion: 3.3.19-beta2
```
Comment (stephengillie) [inline manifests/g/GitHub/GitHubDesktop/Beta/3.4.1-beta1/GitHub.GitHubDesktop.Beta.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.3.19-beta1
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Automatic Validation ended with:
>  Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 3.3.19-beta2 File: GitHub.GitHubDesktop.Beta.locale.en-US.yaml

File: manifests/g/GitHub/GitHubDesktop/Beta/3.4.1-beta1/GitHub.GitHubDesktop.Beta.locale.en-US.yaml (hunk 1 above)
Comment (a-mnich) [reply manifests/g/GitHub/GitHubDesktop/Beta/3.4.1-beta1/GitHub.GitHubDesktop.Beta.locale.en-US.yaml]: Fixed. Copy paste error of myself as there's currently a bug in the automatic pipeline

**Comment by stephengillie** [comment]
Hi @damn-good-b0t,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 883.)

**Pull request closed without merging**